pkg/controllers/nodeclaim/termination/suite_test.go

```diff
@@ -18,6 +18,8 @@ package termination_test
 
```

Comment:
How I would think about the test cases that we now need to ensure:
1. Handle providerID not found
2. Remove the finalizer and terminate the NodeClaim once instance is gone - maybe ensure the InstanceTerminating status condition is there
3. Don't remove the finalizer and terminate the NodeClaim if the instance is still around - ensure the InstanceTerminating status condition exists